### DISCUSSION TITLE
.NET Standard 2.0

### DISCUSSION THREAD

**Discussion by ptrkbmn** [thudugala/Plugin.LocalNotification#180] [Q&A] [answered]
Is there any specific reason why the support for .NET Standard 2.0 was dropped? Do those classes really need .NET Standard 2.1 features? If I modify the target frameworks to netstandard2.0 the project still compiles.

**Comment by thudugala** [comment]
@ptrkbmn planning to use NET Standard 2.1 language features.

**Comment by InquisitorJax** [comment]
@thudugala a big blocker here is for those of us that are targeting UWP, upgrading our common libraries to .NET standard 2.1 is not an option: https://stackoverflow.com/questions/58070082/using-net-standard-2-1-with-uwp
which sadly means that even though I don't need notifications for UWP (yet), it means:
1) I can't use your plugin
2) Future UWP support for this plugin is not currently possible.

**Comment by thudugala** [comment]
@InquisitorJax and @ptrkbmn will convert to NET Standard 2.0, in next version

**Comment by thudugala** [comment, marked as answer]
Added NET Standard 2.0 in and after version 8.0.1
